cmd/juno/cli/root.go

```diff
@@ -138,11 +199,62 @@ var (
 	}
 )
 
+func handleConfig(configParam, flag, envVarName string, t int) error {
```

Comment:
what is t?

Reply:
Added relevant comments for the arguments.